pkg/apiserver/archive/experiment.go

```diff
@@ -55,6 +56,8 @@ func Register(r *gin.RouterGroup, s *Service) {
 
 	// TODO: add more api handlers
 	endpoint.GET("", s.listExperiments)
+	endpoint.GET("/detailSearch", s.experimentDetailSearch)
```

Comment:
Can we use a better endpoint name?

Reply:
I have no idea. Any better suggestions about the name?

Reply:
How about /detail/search?